### DISCUSSION TITLE
Faster switching between projects

### DISCUSSION THREAD

**Discussion by jwerre** [community/community#15799] [Projects and Issues]
There is no way to switch between projects. In fact, once you're in a project there's almost no navigation at all as far as I can tell.  You can click on your profile image in the top right then **Your Projects** but all my projects are under my Organization. As it is I have to do something like this **1. Profile Photo** > **2. Your Organizations** > **3. My Org** > **4. Projects** > **5. My Project**. That's 5 clicks to switch between projects.

**Comment by sunng87** [comment]
Once becoming a heavy user of Projects, I realize this is a pain point to me too. I have three active projects and switching has been a pain point.
